### ISSUE TITLE
ImGui error when compiling

### ISSUE THREAD

**Issue by LilFlame** [SilentProjects/shit-pasted-hack#1]
I have an error when i compile this source:
Can not open source file: imgui_dx9.cpp/imgui.cpp/imgui_draw.cpp.
Normally, i know how to include files, but i dont see any imgui file in this source

**Comment by superyor** [comment]
Same, im clueless, they are in the editor but not in any folder.

**Comment by TXIuTnVsbA** [comment]
its not use in this projects,you can delete it

**Comment by xXTurnerLP** [comment]
Yes, I can confirm that by removing the entire filter you can compile!
